injected/integration-test/test-pages/webcompat/config/enumerate-devices-api.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Any reason you have both this and device-enumeration.json?

Reply:
Probably file rename let me delete :)